tests/Predis/Transaction/MultiExecTest.php

```diff
@@ -479,7 +489,8 @@ public function testHandlesStandardExceptionsInBlock()
             // NOOP
         }
 
-        $this->assertNull($responses, $expected);
+        $this->assertNull($responses);
+        $this->assertIsArray($expected);
```

Comment:
I admit I'm not sure what happens here in my original implementation of this test when I wrote that `$this->assertNull($responses, $expected);` line. I mean if the assertion fails due to a bug or a regression, it prints the contents of `$expected` in the summary of the failed test... but I can't think of a reason why this could be useful in any way. I'd investigate this before changing the code here, could you please revert this change or does PHPUnit 8 break with an error?

Anyway testing for `$this->assertIsArray($expected);` is useless as it's defined on top of the test body and can't be changed in any way by the test since it's not passed by reference anywhere.

Reply:
By adding the line:

```
$this->assertNull($responses, $expected);
```

it throws the following error:

```
TypeError: Argument 2 passed to PHPUnit\Framework\Assert::assertNull() must be of the type string, array given, called in /code/tests/Predis/Transaction/MultiExecTest.php on line 492
```

However, if i change the line to the following, the test passes:

```
$this->assertNull($responses, json_encode($expected));
```

Reply:
OK then just change it to:

```php
$this->assertNull($responses, 'Transaction is expected to return NULL when an exception is thrown in the callable block');
```